src/components/Footer.js

```diff
@@ -136,15 +78,51 @@ export const query = graphql`
         }
       }
     }
+    allMarkdownRemark(
+      filter: { frontmatter: { template: { eq: "footer-nav" } } }
+    ) {
+      edges {
+        node {
+          frontmatter {
+            email_link {
+              label
+              email
+            }
+            nav_dropdowns {
+              title
+              dropdown_items {
+                label
+                is_external_link
+                url
+              }
+            }
+            copyright_text
+            legal_links {
+              label
+              url
+            }
+          }
+        }
+      }
+    }
   }
 `
 
 const Footer = () => (
   <StaticQuery
     query={query}
     render={(data) => {
-      const { halfCircle } = data
-      return <FooterTemplate images={{ halfCircle }} />
+      const { halfCircle, allMarkdownRemark } = data
+      const { frontmatter } = allMarkdownRemark.edges[0].node
+      return (
+        <FooterTemplate
+          images={{ halfCircle }}
```

Comment:
Why isn't this a background thing managed in CSS, out of curiosity?

Reply:
I wanted to use a background image instead, but you can't rotate background images as far as I can tell? It would be easier css-wise and one less query, but then we'd have to include a second version of the image with it rotated - and I thought performance-wise it would be better to download a single image asset instead of two. I almost did it, since the additional rules we have to add for the rotation are annoyingly finicky. 😆

Reply:
Wondering if this is achievable with a :before pseudoelement rotated with a background. Anyway, not a huge deal.